.devcontainer/installJava.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
```

Comment:
Is this script necessary now that we have devcontainer-feature?

Reply:
I have removed this script file.